### DISCUSSION TITLE
EpiNow2 crashes without meaningful error message

### DISCUSSION THREAD

**Discussion by fabsig** [epiforecasts/EpiNow2#292] [General]
I am running EpiNow2 version 1.3.2 on Windows 10 with R version 4.0.2.

When I call the `epinow()` function, my R session crashes without giving any meaningful error message (see screenshot below). Below is also the code that I am running as well as the log output. Any explanation for this?

![epinow2_crash](https://user-images.githubusercontent.com/32902180/147064641-f1fd2700-07e9-4210-808d-b7a41aee1ab1.jpg)

Code called:
```
library(EpiNow2)
reporting_delay <- list(
    mean = convert_to_logmean(2, 1), mean_sd = 0.1,
    sd = convert_to_logsd(2, 1), sd_sd = 0.1,
    max = 10
)
generation_time <- get_generation_time(disease = "SARS-CoV-2", source = "ganyani")
incubation_period <- get_incubation_period(disease = "SARS-CoV-2", source = "lauer")
reported_cases <- data.frame(date=data$date, confirm=data$confirm)
reported_cases$date <- as.Date(reported_cases$date)
estimates <- epinow(reported_cases = reported_cases,
                    generation_time = generation_time,
                    delays = delay_opts(incubation_period, reporting_delay),
                    rt = rt_opts(prior = list(mean = 2, sd = 0.2)),
                    stan = stan_opts(cores = 4), CrIs = c(0.95))
```

Log file:
```
DEBUG [2021-12-22 09:42:03] epinow: Running in exact mode for 2000 samples (across 4 chains each with a warm up of 250 iterations each) and 70 time steps of which 7 are a forecast
```

**Comment by seabbs** [comment]
Hi @fabsig 

Do you have a complete reprex that recreates this behaviour? Are you able to run the examples given in the code? 

Just a note that your prior for Rt (mean of 2 with a standard deviation of 0.2) seems high for most settings I am aware of.

Sam

**Comment by seabbs** [comment]
These kind of Rstudio bugs are usually ram related. Have you looked at your usage to see if this is the issue?
